seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/TablePath.java

```diff
@@ -65,8 +65,7 @@ public static TablePath of(String fullName, boolean schemaFirst) {
         if (paths.length == 3) {
             return of(paths[0], paths[1], paths[2]);
         }
-        throw new IllegalArgumentException(
-                String.format("Cannot get split '%s' to get databaseName and tableName", fullName));
+        return of(null, null, fullName);
```

Comment:
If topic is `test.multiple.point`. It would be convert to `database=test,schema=multiple,table=point`. There are different numbers of dot that affect the splitting of the content, it's unreasonable.